src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/navbar-logo.html

```diff
@@ -1,19 +1,31 @@
-{% if logo %}
 {% if not theme_logo_link %}
-<a class="navbar-brand" href="{{ pathto(root_doc) }}">
-  <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
-</a>
-{% elif theme_logo_link[:4] == 'http' %}
-<a class="navbar-brand" href="{{ theme_logo_link }}">
-  <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
-</a>
+  {% set href = pathto(root_doc) %}
+{% elif theme_logo_link[:4] == "http" %}
+  {% set href = theme_logo_link %}
 {% else %}
-<a class="navbar-brand" href="{{ pathto(theme_logo_link) }}">
+  {% set href = pathto(theme_logo_link) %}
+{% endif %}
+
+{% if theme_dark_logo and theme_light_logo %}
+
```

Comment:
Maybe here we could do a quick check for if `html_logo` is specified, but not `theme_light_logo`. So something like

```
{% if theme_dark_logo %}
{% if not theme_light_logo and logo %}
{% set theme_light_logo = logo %}
{% endif %}
...
{% endif %}

Reply:
I think even better would be to raise a warning or helpful error, but I'm not sure how to do that from within Jinja

Reply:
I don't think you can raise a warning from jinja. I decided to go as followed: 
- check if one of the three is set
- always set 2 images (even if it's the same logo)